src/Tax.ts

```diff
@@ -75,7 +75,7 @@ export const calculatetax = () => {
         exponent *= 0.5
     }
     G['maxexponent'] = Math.floor(275 / (Decimal.log(1.01, 10) * exponent)) - 1
-    const a2 = Math.min(G['maxexponent'], Math.floor(Decimal.log(G['produceTotal'].add(1).toNumber(), 10)));
+    const a2 = Math.min(G['maxexponent'], Math.floor(Decimal.log(G['produceTotal'].add(1), 10)));
```

Comment:
Pushed in a828f16b3e467bfb0fbd98d6252dcf7e48ee61ed, didn't see it here. My bad.

Reply:
No worries :) Glad it's fixed!